docs/patch.md

```diff
@@ -123,6 +123,39 @@ Here's a minimal example showing how to clear a given path:
 patch.clear('custom.pets');
 ```
 
+### remove
+
+This method removes all values that match in a collection.
+
+The following table shows how the operation behaves in different scenarios:
+
+Current Value                     | Given Value       | Result
+----------------------------------|-------------------|-------------------
+`['a', 'b', 'c']`                 | `'a'`             | `['b', 'c']`
+`['a', 'b', 'c', 'c']`            | `'c'`             | `['a', 'b']`
+`['a', 'b', 'c']`                 | `'d'`               | `['a', 'b', 'c']`
+`[]`                              | `'foo'`           | `[]`
+`['foo', {'qux': 'quux'}, 'bar']` | `{'qux': 'quux'}` | `['foo', 'bar']`
+
+
+#### Signature
+
+The `remove` method has the following signature:
+
+```ts
+patch.remove(path: string, value: JsonValue)
+```
+
+The return is the path instance itself to allow operation chaining.
```

Comment:
I've just noticed a typo in all methods... can you fix it for me, please? The correct here is this:

```
The return is the `Patch` instance itself to allow operation chaining.
```